lib/vistas/paqueteForm.dart

```diff
@@ -1,48 +1,445 @@
-        title: Text(title),
+  void add() {
+    pesoController.text = (peso += 1).toString();
+  }
+
+  void remove() {
+    if (peso - 1 >= 0.0) pesoController.text = (peso -= 1).toString();
+  }
+
+  InputDecoration returnInputDecoration(String hintText) {
+    return InputDecoration(
+      suffixIcon: hintText.startsWith("Punto") ? Icon(Icons.location_on) : null,
+      hintText: hintText,
+      hintStyle: TextStyle(color: TransportifyColors.primarySwatch),
+      filled: true,
+      fillColor: Colors.white,
+      contentPadding: EdgeInsets.fromLTRB(20.0, 15.0, 20.0, 15.0),
+      border: OutlineInputBorder(borderRadius: BorderRadius.circular(10.0)),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Form(
+      key: _formKey,
+      child: Scaffold(
+        appBar: AppBar(
+          // Here we take the value from the MyHomePage object that was created by
+          // the App.build method, and use it to set our appbar title.
+          title: Text(PaqueteForm.title),
+          backgroundColor: TransportifyColors.primarySwatch,
+          elevation: 0.0,
+        ),
         backgroundColor: TransportifyColors.primarySwatch,
-        elevation: 0.0,
-      ),
-      backgroundColor: TransportifyColors.primarySwatch,
-      body: Center(
-        child: ListView(
-          padding: EdgeInsets.only(left: 24.0, right: 24.0),
-          children: <Widget>[
-            SizedBox(height: 48.0),
-            nombrePaquete,
-            SizedBox(height: 8.0),
-            SizedBox(height: 24.0),
+        body: Center(
+          child: ListView(
+            padding: EdgeInsets.only(left: 15.0, right: 15.0),
+            children: <Widget>[
+              SizedBox(height: 10.0),
+              TextFormField(
+                maxLines: 1,
+                keyboardType: TextInputType.text,
+                autofocus: false,
+                style: TextStyle(color: TransportifyColors.primarySwatch),
+                decoration: returnInputDecoration("Nombre"),
+                controller: nombreController,
+                validator: (value) {
+                  if (value.isEmpty)
+                    return 'Nombre';
+                  else
+                    return null;
+                },
+              ),
+              SizedBox(height: 20.0),
+              Row(
+                children: <Widget>[
+                  Expanded(
+                    child: TextFormField(
+                      maxLines: 1,
+                      keyboardType: TextInputType.number,
+                      autofocus: false,
+                      style: TextStyle(color: TransportifyColors.primarySwatch),
+                      decoration: returnInputDecoration("Peso"),
+                      onChanged: (text) {
+                        peso = double.parse(text);
+                      },
+                      controller: pesoController,
+                      validator: (value) {
+                        if (value.isEmpty)
```

Comment:
Para el campo 'Peso', también se debe comprobar que el valor es positivo.

He mirado cómo lo haces en alto, ancho y largo (donde sí está bien implementado) y he creado la sugerencia en base a esto.

```suggestion
                        if (value.isEmpty || double.parse(value) <= 0)
```